plugins/frontend/backstage-plugin-argo-cd/src/components/ArgoCDDetailsCard.tsx

```diff
@@ -165,47 +189,71 @@ const OverviewComponent = ({
     {
       title: 'Name',
       highlight: true,
+      field: 'name',
       render: (row: any): React.ReactNode =>
         detailsDrawerComponent(row, getBaseUrl(row)),
+      customSort: (a: any, b: any) =>
```

Comment:
Do need custom sort functions for strings ? IIRC you can use dot notation in the field ? e.g. field: 'metadata.name' ? string sort should work out of the box in that case ? maybe I'm missing something

Reply:
I defined `customSort` because I couldn't make it work. After some digging I found [this](https://github.com/material-table-core/core/issues/826) issue and I applied solution similar to suggested in [this](https://github.com/material-table-core/core/issues/826#issuecomment-1924008494) comment.

Reply:
👍